tools/intrinsic_db/advsimd_classification.csv

```diff
@@ -4389,4 +4577,55 @@ vbfmlaltq_f32	Vector arithmetic|Multiply|Multiply-accumulate
 vbfmlalbq_lane_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
 vbfmlalbq_laneq_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
 vbfmlaltq_lane_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
-vbfmlaltq_laneq_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
\ No newline at end of file
+vbfmlaltq_laneq_f32	Scalar arithmetic|Vector multiply-accumulate by scalar
```

Comment:
I think we need to put more effort into intergerating this with the classification scheme.  This information is used for the instruction group on https://developer.arm.com/architectures/instruction-sets/intrinsics/ .  So new top-level classifications should not be added lightly.  And specific element types should not be mentioned, since those are part of the separate “return base type” and “element bit size” selectors.